### ISSUE TITLE
I/O mode counter-intiuitive

### ISSUE THREAD

**Issue by khthe8th** [zeroneye/Powah#208]
Version:
    Minecraft: 1.16.5
    Forge: 36.2.20
    Powah: 2.3.16

Issue:
On most/all pieces of Powah energy the I/O options are "receive", "extract", and "rec/ext". The issue is receive and extract mean the same thing, both would be a pull of energy. However, in the game receive is the actual pull and extract is a push. Which is not intuitive at all. 

Can these wordings be changed? The opposite of receive would be send.

**Comment by andrewsf** [comment]
"Receive" from another location, i.e., the Ender network.
"Extract" from a container or block, i.e., something locally attached.

It's confusing, but there is a logic to it. I don't agree that the two words mean the same thing -- one is active and one is effective/passive. You can receive through no action of your own ("be pushed to"), unlike extraction ("pull from").

Yes, the words should be changed if (the real) owmii ever reappears and works on these projects again.
I recommend to raise an issue on [Technici4n's fork](https://github.com/Technici4n/Powah), which is actively maintained.

**Comment by BrightCandle** [comment]
I got confused by this a number of times. Most other blocks tend to use in and out in reference to the block and I think that at least would be a more consistent naming with other tech mods like Thermal.

**Comment by LZeugirdor** [comment]
I agree, Input and Output is easier for me to understand personally. Would help if there was color coding like most other mods do with blue input and orange output.
